components/skeleton/doc/index.zh-CN.md

```diff
@@ -0,0 +1,42 @@
+---
+category: Components
+subtitle: 加载占位图
+type: Data Entry
+title: Skeleton
+cols: 1
+---
+
+缓解用户等待的焦虑，建立用户的心理预期，提升加载过程的流畅度。
+
+## 何时使用
+
+- 网络较慢，需要长时间等待加载处理的情况下
+- 图文信息内容较多的列表/卡片中
+
```

Comment:
Could you please add the other demo? See: https://deploy-preview-11226--ant-design.netlify.com/components/skeleton-cn/.

It would be good if we sync to ant design react.

Reply:
thx, I just sent a commit